packages/dotnet-sdk/README.md

```diff
@@ -135,6 +137,17 @@ private void printLog(LogLevel level, string message)
 }
 ```
 
+### Upgrade from 0.3.0-rc to 0.4.0 (For projects created by Visual Studio 2022 17.1 Preview toolkit)
+If there is an existing project created in VS2022 17.1 Preview, you can use the following steps to upgrade:
+- In `appsettings.{Environment}.json` file:
+1. Add `OAuthAuthority` under `TeamsFx:Authentication` and copy the value from `OAUTH_AUTHORITY`.
+2. Remove the line `"SimpleAuthEndpoint": "https://localhost:44357/"`.
```

Comment:
Maybe this line could be 
Remove the line `"SimpleAuthEndpoint": "https://localhost:port/"`.